### PR TITLE
Update Create an eCommerce landing page

### PR DESCRIPTION
**Pull request by laurosilvacom**
- Updates the course landing page by adding the additional learning resources (articles, talk)
- articles have consistent covers

CleanShot 2021-02-16 at 01 17 03@2x

### CODE REVIEW

File: src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx [hunk 1]
```diff
@@ -586,4 +614,123 @@ const Part: FunctionComponent<{
...
+
+const Articles = ({articles, children}) => {
+  return (
+    <div className="bg-gray-50 dark:bg-gray-900 -mx-5 pt-24 pb-40 xl:px-0 px-5 text-white pb-12">
+      <div className="max-w-screen-lg mx-auto">
+        <div className="mb-4 uppercase font-medium tracking-wide text-sm md:text-left text-center text-blue-600">
+          Build Boyond this project
+        </div>
+        <h2 className="sm:text-2xl text-3xl md:text-left font-semibold text-center leading-tighter pb-4 dark:text-gray-200 text-gray-900">
+          Addtional Learnign Resources
```
Comment (zacjones93) [inline src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx:626]:
```suggestion
          Additional Learning Resources
```

**Review by zacjones93 — CHANGES_REQUESTED** [review]

File: src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx [hunk 2]
```diff
@@ -586,4 +614,123 @@ const Part: FunctionComponent<{
     </div>
   )
 }
+
+const Articles = ({articles, children}) => {
```
Comment (joelhooks) [inline src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx:618]:
```suggestion
const Articles: React.FC<{articles: any}> = ({articles, children}) => {
```

You can't use `any` implicitly and have to be explicit about it like this. Even better to actually type it.

Comment (joelhooks) [reply src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx:618]: note that `children` is typed in `React.FunctionComponent` so no need to double up on it.

File: src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx [hunk 3]
```diff
@@ -586,4 +614,123 @@ const Part: FunctionComponent<{
...
+            </div>
+          </div>
+        </div>
+        {children}
+      </div>
+    </div>
+  )
+}
+
+function FeaturedVideoCard({video}) {
```
Comment (joelhooks) [inline src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx:675]:
```suggestion
const  FeaturedVideoCard: React.FC<{video: any}> = ({video}) => {
```

Welcome to TypeScript

File: src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx (hunk 2 above)
Comment (laurosilvacom) [reply src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx:618]: This makes sense. I'll make sure to be explicit about it.

Comment (joelhooks) [reply src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx]: more to the point TypeScript will make sure youre explicit about it lol 😅

**Review by zacjones93 — APPROVED** [review]

**Pull request merged**